libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -590,7 +606,8 @@ AP_GPS_UBLOX::_parse_gps(void)
     if (_class == CLASS_ACK) {
         Debug("ACK %u", (unsigned)_msg_id);
 
-        if(_msg_id == MSG_ACK_ACK) {
+        // a nack has the same effect as an ack
+        if(_msg_id == MSG_ACK_ACK || _msg_id == MSG_ACK_NACK) {
```

Comment:
Why would we accept nack's here? That seems like exactly what we don't want to accept, as that means it refused our desired change.